lighthouse-cli/test/smokehouse/seo/expectations.js

```diff
@@ -49,6 +52,10 @@ module.exports = [
         score: false,
         displayValue: '403',
       },
+      'anchor-text': {
+        score: false,
+        displayValue: '3 anchors found',
```

Comment:
Checking length SGTM, but checking all contents gets very verbose (showing just one of three items):
```
        details: {
          items: {
            length: 3,
            0: {
              0: {
                text: 'https://example.com/',
              },
              1: {
                text: 'click this',
              }
            },
          },
        },
```
I'd leave that to unit tests (in fact, since we are getting rid of `extendedInfo`, I'll have to use `details` in unit tests).

Reply:
> I'd leave that to unit tests

SGTM